scripts/check-migrations.py

```diff
@@ -39,7 +39,7 @@ def _get_changes(globs: Sequence[str], workdir: str, to: str) -> str:
             "diff",
             "--diff-filter=AM",
             "--name-only",
-            to,
+            f"{to}...",
```

Comment:
We want make the diff relative so that we avoid showing files in the diff that wouldn't appear after a rebase/merge. This is the diff that appears in PRs on the github UI.

Reply:
I'm having trouble understanding as well, the diff we see in the GitHub UI is exactly what is changing between the PR's branch and the master branch. 

> avoid showing files in the diff that wouldn't appear after a rebase/merge

Avoid showing files in the diff that wouldn't appear after merging master into the PR? Isn't that how it already works?

Reply:
i think this change makes sense

when a PR is based off of an old commit from master, `git diff origin/master` shows changes that are not related to the PR and rather are the inverse of changes that have happened on master since the PR has been opened

you can try it out on this PR itself. new changes have happened on master, and `git fetch origin && git diff origin/master` will show you changes unrelated to the PR

the github PR diff is also `git diff origin/master...` or something similar